<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uavposdialog</class>
 <widget class="QDialog" name="uavposdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>文件路径</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>载入曝光点文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePosFile"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpenPos">
          <property name="text">
           <string>浏览...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>文件格式</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>7</number>
        </property>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>从文件开头起忽略的行数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rowCounter">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>The number of lines to discard from the beginning of the file</string>
            </property>
            <property name="statusTip">
             <string>The number of lines to discard from the beginning of the file</string>
            </property>
            <property name="whatsThis">
             <string>The number of lines to discard from the beginning of the file</string>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxUseHeader">
            <property name="toolTip">
             <string>Field names are read from the first record. If not selected then fields are numbered</string>
            </property>
            <property name="statusTip">
             <string>Field names are read from the first record. If not selected then fields are numbered</string>
            </property>
            <property name="whatsThis">
             <string>Field names are read from the first record. If not selected then fields are numbered</string>
            </property>
            <property name="text">
             <string>首行包含字段名称</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>相片编号</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>记录选项</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>字段选项</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QCheckBox" name="chkClipSpace">
            <property name="text">
             <string>裁剪字段前后的空格</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxSkipEmptyFields">
            <property name="text">
             <string>忽略空字段</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkDMSformat">
            <property name="text">
             <string>度分秒格式坐标</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="chkComma">
            <property name="text">
             <string>逗号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkTab">
            <property name="text">
             <string>制表符</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkSpace">
            <property name="text">
             <string>空格</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkColon">
            <property name="text">
             <string>冒号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkSemicolon">
            <property name="text">
             <string>分号</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>格式定义</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>相片编号</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>横坐标</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>纵坐标</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>绝对行高</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Omega</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Phi</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Kappa</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>相机标识</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QComboBox" name="cmb_1"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_2"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_3"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_4"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_5"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_6"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_7"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_8"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lePosFile</tabstop>
  <tabstop>btnOpenPos</tabstop>
  <tabstop>chkComma</tabstop>
  <tabstop>chkTab</tabstop>
  <tabstop>chkSpace</tabstop>
  <tabstop>chkColon</tabstop>
  <tabstop>chkSemicolon</tabstop>
  <tabstop>rowCounter</tabstop>
  <tabstop>cbxUseHeader</tabstop>
  <tabstop>chkClipSpace</tabstop>
  <tabstop>cbxSkipEmptyFields</tabstop>
  <tabstop>chkDMSformat</tabstop>
  <tabstop>cmb_1</tabstop>
  <tabstop>cmb_2</tabstop>
  <tabstop>cmb_3</tabstop>
  <tabstop>cmb_4</tabstop>
  <tabstop>cmb_5</tabstop>
  <tabstop>cmb_6</tabstop>
  <tabstop>cmb_7</tabstop>
  <tabstop>cmb_8</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>uavposdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>uavposdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
